go/libkb/logout.go

```diff
@@ -97,16 +123,16 @@ func (m MetaContext) logoutSecretStore(username NormalizedUsername, killSecrets
 
 // LogoutSelfCheck checks with the API server to see if this uid+device pair should
 // logout.
-func (m MetaContext) LogoutSelfCheck() error {
-	g := m.G()
+func (mctx MetaContext) LogoutSelfCheck() error {
+	g := mctx.G()
```

Comment:
will this pass lint? i think there is some rule where all receivers for given type in a file need to be named the same